Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 import Material
 
-extension ResetPasswordViewController {
+extension ResetPasswordViewController:UITextFieldDelegate {
```

Comment:
Follow the Swift guidelines mentioned in the README. There should be a space after `:`